### PR TITLE
feat: move over to kts file if you want too

### PR DESCRIPTION
**Pull request by The-Repo-Club**
i'm not sure if you want these or not or if your happy with using groovy but they are here if you do.

### CODE REVIEW

**Comment by The-Repo-Club** [comment]
may need to remove the 3 non .kts for it to build with the kts versions though :) but this is a starter

**Comment by jooooscha** [comment]
What benefits does this have?

**Comment by The-Repo-Club** [comment]
Tbh none except preference also [just check this ](https://youtu.be/ErdH9mQySQQ) this is why I switched

File: app/src/main/java/app/olaunchercf/ui/SettingsFragment.kt [hunk 1]
```diff
@@ -371,7 +371,7 @@ class SettingsFragment : Fragment() {
                 modifier = Modifier
                     .align(Alignment.End)
                     .padding(10.dp, 5.dp),
-                text = "Version: ${requireContext().packageManager.getPackageInfo(requireContext().packageName, 0).versionName}",
+                text = "Version: ${BuildConfig.VERSION_NAME}",
```
Comment (The-Repo-Club) [inline app/src/main/java/app/olaunchercf/ui/SettingsFragment.kt:374]: also removed a deprecated function

**Pull request closed without merging**